### DISCUSSION TITLE
ProductBoard integration

### DISCUSSION THREAD

**Discussion by esethna** [mattermost/community#34] [Practical Ideas]
Currently email is the way to get notified of new activity in Productboard. As much as possible I try to cut out email from my daily workflows, so similar to Andrew's idea to get a Figma integration, we should do the same for Productboard (and really all of our tools)

I want notifications when:

- I'm assigned to a new insight or feature
- Comments on any insight or feature or portal that you own
- I'm @mentioned in comments

**Comment by chetanyakan** [comment]
@esethna What about https://github.com/mattermost/community/discussions/96 to use Mattermost for checking your emails?

**Reply by esethna** [reply]
I'd like to take email out entirely. My preference would be like a github bot where I could get to zero inbox in the RHS after viewing notifications.
